### ISSUE TITLE
Audio - récompenses

### ISSUE THREAD

**Issue by ElSenorDuvivier** [myriagonofficial/alpha#39]
Voici les récompenses audio en fichier mp3 (destinés au joueur débloquant des trophées)

J'ai mis quelques métadonnées dans les fichiers mp3.

Tout ceci se trouve dans un dossier dédiés dans le drive :
https://drive.google.com/open?id=1aG1hhAo3VhFLvWTqh4K--VxJn5ikP6F7

**Comment by sylvainpolletvillard** [comment]
C'est quoi les trophées ? Les achievements tu veux dire ? Pourquoi il y a un fichier par chapitre alors qu'on le découvre à la fin ? Je comprends pas :(

**Comment by ElSenorDuvivier** [comment]
Oui, si j'ai bien compris chaque achievement ( par exemple bâtisseur) offre
potentiellement un bonus ( par exemple une image à télécharger). Dans le
cas des fichiers audio de récompense /achievement, je pensais à un lien qui
permet de télécharger l'un des fichiers audio en guise de récompense, à la
manière de l'image qu'on peut télécharger quand on a l'achievement
bâtisseur.

**Comment by sylvainpolletvillard** [comment]
Ah OK je vois c'est des récompenses d'achievement. Dans ce cas je laisserais Romain les attribuer aux différents achievements, je ne sais pas ce qu'il a en tête

**Comment by ElSenorDuvivier** [comment]
yes perfect !!!
